bin/cibuild.sh

```diff
@@ -20,7 +20,7 @@ main() {
     master)
       echo "Building with production jekyll config"
 
-      bundle exec jekyll build --destination ./_site/$CF_PATH --baseurl /$CF_PATH
+      bundle exec JEKYLL_ENV=production jekyll build --destination ./_site/$CF_PATH --baseurl /$CF_PATH
```

Comment:
This approach should work, but you have to put the `JEKYLL_ENV=production ` at the very start of the line

Reply:
Hey @gordcorp so with this approach it won't block robots on production?

Reply:
@joolswood correct it wont.

Reply:
Awesome, cheers